.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+rvm:
+  - 1.9.3
```

Comment:
I'd remove this version - https://www.ruby-lang.org/en/news/2014/01/10/ruby-1-9-3-will-end-on-2015/